meshapp/src/components/password-forms/forgot-password-form.tsx

```diff
@@ -0,0 +1,103 @@
+import { useForm } from "react-hook-form";
+import { Button, Grid, Stack, Typography, TextField, Box } from '@mui/material';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import React, { useState } from "react";
+import { axiosInstance } from "../../config/axiosConfig";
+
+const theme = createTheme({
+    palette:{
+      primary: {
+        main: '#68D391'
+      },
+    },
+    typography: {
+        fontFamily: 'cocogoose, sans-serif'
+    },
+    components: {
+      MuiButton: {
+        styleOverrides: {
+          root: {
+            borderRadius: 25,
+            width: '75%',
+            height: '4.5rem',
+            fontSize: '1.5rem',
+          },
+        },
+      },
+    }
+  });
+
+export const PasswordReset = () => {
+
+    const passwordResetConfirmedEndpoint = 'user/reset/confirmed'
+
+    // input validation on password reset form 
+    const { register, handleSubmit, formState: { errors } } = useForm();
+
+    // stores password inputs in state
+    const [formData, setFormData] = useState({
+        password: '',
+        confirmedPassword: '',
+
+    });
+
+    // updates form data in state when input fields detect change
+    const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setFormData({
+        ...formData,
+        [e.target.name]: e.target.value,
+        })
+    }
+
+    // submit user data to backend
+    const onSubmit = () => {
+        if (formData.password == formData.confirmedPassword){
+            const res = axiosInstance.post(passwordResetConfirmedEndpoint,{
+              newpassword: formData.confirmedPassword
+          }).then(
+            (res) => {
+              console.log(res.data)
+            }
+          )
+        } else {
+          console.log("Passwords must match!")
+        }
+
+    }
+
+    // TODO: https://github.com/LetsMesh/Site/issues/255
+
+    return (
```

Comment:
I understand that cocogoose was applied to all text, but in the figma only certain elements such as headings use cocogoose, Elements such as input labels should use a sans serif font.